_data/news.yml

```
@@ -13,6 +13,9 @@
 - name: Deliverable D1 published (requirements analysis)
   date: 2025-03-31
   description: "The first project deliverable D1 is published: [Requirements analysis based on the two TeSS instances from ELIXIR and PaNOSC doi:10.5281/zenodo.15120731](https://doi.org/10.5281/zenodo.15120731)"
+- name: Mention at WLCG/HSF Workshop 2025 in Paris, France
```

```suggestion
- name: Presentation at WLCG/HSF Workshop 2025 in Paris, France
```
I think the word "mention" in the title here sounds like we are diminishing/underselling it. You were on the programme and giving a presentation, even if it wasn't a presentation about mTeSS-X exclusively.